src/Symfony/Component/Debug/Exception/FatalThrowableError.php

```diff
@@ -18,8 +18,12 @@
  */
 class FatalThrowableError extends FatalErrorException
 {
+    private $throwable;
+
     public function __construct(\Throwable $e)
     {
+        $this->throwable = $e;
```

Comment:
We should only keep the class name, not the instance, to make this lighter

Reply:
We then would need to keep the original message as well, because `FatalThrowableError` tinkers with it. But yes, I can change that.

Reply:
Okay, I'll remove it.